xarray/core/variable.py

```diff
@@ -1247,6 +1247,19 @@ def copy(self, deep=True):
         return type(self)(self.dims, self._data, self._attrs,
                           self._encoding, fastpath=True)
 
+    def equals(self, other, equiv=None):
+        # if equiv is specified, super up
+        if equiv:
```

Comment:
I'd prefer `equiv is not None`, which is slightly more idiomatic for checking default values (I'd rather not rely on the truthiness of arbitrary functions/callables)